### PR TITLE
ensure proper rpms installed

### PR DESCRIPTION
**Pull request by tchughesiv**

### CODE REVIEW

File: Dockerfile.rhel7 [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # This image is the base image for all OpenShift v3 language Docker images.
-FROM rhel7
+FROM registry.access.redhat.com/rhel7
```
Comment (bparees) [inline Dockerfile.rhel7]: this value("rhel7") is presently a necessary artifact of how we do our internal builds of this image.

Comment (tchughesiv) [inline Dockerfile.rhel7]: gotcha, reverted w/ https://github.com/sclorg/s2i-base-container/pull/118/commits/ef46e0ebb849d5a79cf26b4ff0daa7d441178d78

**Comment by bparees** [comment]
thanks, i'll look into changing our internal behavior so we can make this align in the future.

[test]

**Comment by bparees** [comment]
sorry I was thinking this was on a different repo.  @hhorak you guys ok w/ these changes to the enabled repo list?

**Comment by omron93** [comment]
+1

Other images using `s2i-base` as a base image would benefit from this too.

**Comment by hhorak** [comment]
Makes sense to me as well. Merging.

**Pull request merged**